index.html

```diff
@@ -32,40 +32,295 @@
+              >
+                <i class="fa-brands fa-twitter"></i>
+              </a>
+            </li>
+            <li class="icon">
+              <a
+                href="https://www.linkedin.com/in/dennis-every/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fa-brands fa-linkedin-in"></i>
+              </a>
+            </li>
+            <li class="icon">
+              <a
+                href="https://dennis-every.medium.com/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fa-brands fa-medium"></i>
+              </a>
+            </li>
+            <li class="icon">
+              <a
+                href="https://github.com/dennis-every"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fa-brands fa-github"></i>
+              </a>
+            </li>
+            <li class="icon">
+              <a
+                href="https://angel.co/u/dennis-every"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i class="fa-brands fa-angellist"></i>
+              </a>
+            </li>
+          </ul>
         </li>
-        <li class="icon">
-          <a href="https://angel.co/u/dennis-every" target="_blank">
-            <i class="fa-brands fa-angellist fa-lg"></i>
-          </a>
+      </ul>
+    </section>
+    <section id="works-container">
+      <ul id="works">
+        <li class="work-card">
+          <img
+            src="images/work_1.jpg"
+            alt="work_tonic_daily_personalized_reads"
+          />
+          <h2>Tonic</h2>
+          <ul class="experience">
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <span
+            >A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</span
+          >
+          <ul class="technologies">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+        <li class="work-card">
+          <img
+            src="images/work_2.jpg"
+            alt="work_multi-post_stories_daily_personalized_reads"
+          />
+          <h2>Multi-Post Stories</h2>
+          <ul class="experience">
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <span
+            >A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</span
```

Comment:
- [ ] Please for this text use a `p` tag instead of a `span` tag and make sure to do that for all your work sections.